apps/root/src/hooks/earn/useSuggestedStrategies.ts

```diff
@@ -21,7 +21,9 @@ export default function useSuggestedStrategies(
       strategiesWithReward: Strategy[];
     }>(
       (acc, strategy) => {
-        const isSameAsset = getIsSameOrTokenEquivalent(selectedAsset.token, strategy.asset);
+        const isSameAsset =
+          getIsSameOrTokenEquivalent(selectedAsset.token, strategy.asset) ||
+          getIsSameOrTokenEquivalent(selectedReward.token, strategy.asset);
```

Comment:
This is not happening for now, but there could be strategies that don't generate yield in their asset token. We can check that by seeing the APY